### PR TITLE
Addresses #112: sets default to `n` in `hit clear` prompt.

### PR DESCRIPTION
**Pull request by zfnmxt**
I modified `yesOrNo` to take a default `Answer` to make it more robust for future usage.

### CODE REVIEW

File: src/Hit/ColorTerminal.hs [hunk 1]
```diff
@@ -116,18 +117,22 @@ arrow = " ➤  "
 data Answer = Y | N
 
 -- | Parse an answer to 'Answer'
-yesOrNo :: Text -> Maybe Answer
-yesOrNo (T.toLower -> answer )
-    | T.null answer = Just Y
+yesOrNo :: Answer -> Text -> Maybe Answer
```
Comment (vrom911) [inline src/Hit/ColorTerminal.hs]: Could you patch the documentation to this function saying that it also should take a default answer?

Comment (vrom911) [reply src/Hit/ColorTerminal.hs]: Also, probably it's better to name it `yesOrNoWithDefault`?

File: src/Hit/ColorTerminal.hs [hunk 2]
```diff
@@ -116,18 +117,22 @@ arrow = " ➤  "
...
     | answer `elem` ["yes", "y", "ys"] = Just Y
     | answer `elem` ["no", "n"]  = Just N
     | otherwise = Nothing
 
-prompt :: IO Answer
-prompt = do
-    answer <- yesOrNo . T.strip <$> getLine
+yesOrNoText :: Answer -> Text
+yesOrNoText N = boldCode <> "[n]" <> resetCode <> "/y"
+yesOrNoText Y = boldCode <> "[y]" <> resetCode <> "/n"
```
Comment (vrom911) [inline src/Hit/ColorTerminal.hs]: I would prefer to have consistency here. SO, it's better to have
```
[y]/n
y/[n]
```

File: src/Hit/ColorTerminal.hs [hunk 3]
```diff
@@ -116,18 +117,22 @@ arrow = " ➤  "
...
+
+prompt :: Answer -> IO Answer
```
Comment (vrom911) [inline src/Hit/ColorTerminal.hs]: Could you please add docs here as well explaining what `Answer` it expects as the input?

**Comment by zfnmxt** [comment]
Made the request changes; thanks for the review, @vrom911

File: src/Hit/ColorTerminal.hs [hunk 4]
```diff
@@ -115,19 +116,24 @@ arrow = " ➤  "
...
+    | T.null answer = Just def
     | answer `elem` ["yes", "y", "ys"] = Just Y
     | answer `elem` ["no", "n"]  = Just N
     | otherwise = Nothing
 
-prompt :: IO Answer
-prompt = do
-    answer <- yesOrNo . T.strip <$> getLine
+yesOrNoText :: Answer -> Text
+yesOrNoText N = boldCode <> "y"   <> resetCode <> "/[n]"
```
Comment (vrom911) [inline src/Hit/ColorTerminal.hs]: Let's make `[n]` bold instead :slightly_smiling_face:

Comment (zfnmxt) [reply src/Hit/ColorTerminal.hs]: Oops, sorry about that. Missed it when I switched the order. All fixed. :)

Comment (vrom911) [reply src/Hit/ColorTerminal.hs]: No worries!

**Pull request merged**